### DISCUSSION TITLE
Global Game Over Prompt

### DISCUSSION THREAD

**Discussion by ronombao24** [inkle/ink#757] [Writing in ink]
Hello. How to create a global game over stitch that happens when a specific scenario is met?

I have this code fragment that I made, for instance:
`//START OF GAME OVER

//Prints a game over message, then it asks for the player to restart from the current knot.

~game_over=false

->GameOverInit

=GameOverInit

->DONE

=GameOver(state)
~state=game_over
->game_over_message->
->->

=game_over_message
{
    - game_over:
    Your journey comes to an end. Do you want to restart from the current knot?
    *YES
    ~game_over=false
    ->->
    *NO
    GAME OVER!
    ->END
}

//END OF GAME OVER`

But I don't know how to make the "YES" prompt redirect to the current knot that the game over happened. For example, the game over happens at a scenario when the player dies from a fall, then the player will be given an option to restart from the current knot (the_knot) or not.

**Comment by GreenCloversGames** [comment]
You can pass diverts into knots. For example, you could do this - 
```
===gameOver(->camefrom)
You are dead.
*restart
        ->Start
*retry
        ->camefrom
```
Then, when you want to use gameOver, you can say - 

```
===MakeAChoice
*Hug him
You give the villain a hug. He stabs you.
    ->gameOver(->MakeAChoice)
*Shoot him
    You shoot him. 
    ->DONE
```
